draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
       </section>
     </section>
 
+    <section title="SDP Identity Attribute" anchor="sec.sdp-id-attr">
+      <t>
+	    The SDP 'identity' attribute is a session-level attribute that is used
+	    by an endpoint to convey its identity assertion to its peer. The identity
+        assertion value is encoded as a <xref target="RFC4648">Base-64 string</xref>
+      </t>
+      <t>
+	    The procedures in this section are based on the assumption that the identity assertion 
+        of an endpoint is bound to the fingerprints of the endpoint. This does not preclude the 
+        definition of alternative means of binding an assertion to the identity, but such means are 
+        outside the scope of this specification.
+      </t>
+      <t>
+	    The semantics of multiple 'identity' attributes within an offer or answer are undefined.
+	    Implementations SHOULD only include a single identity attribute in an
+	    offer or answer and relying parties MAY elect to ignore all but the first
+	    identity attribute. However, a single identity attribute can convey multiple 
+	    identity assertions.
+      </t>
+      <figure>
+      <artwork type="inline"><![CDATA[
+
+Name: identity
+
+Value: identity-assertion
+
+Usage Level: session
+
+Charset Dependent: no
+
+Default Value: N/A
+
+Syntax:
```

Comment:
I realize the ABNF has been around for awhile but I noticed a few things about it that made me think someone should have a look over it.

Reply:
We did get a review from Paul K back in May.  Those changes appear to have been incorporated.  Why don't we move this document towards Adam and we can ask for a re-review from Paul K during IETF LC.